Streams 1750 - document new REST api paths (#71)
Axway/streams-open-docs: content/en/docs/subscribers/subscriber-webhook.md

> In order to stop the sending of webhook notifications, simply delete the corresponding webhook subscription with following request:

Reviewer: ```suggestion
To stop the sending of webhook notifications, simply delete the corresponding webhook subscription with following request:
```

> In order to get existing subscription, simply do the following GET request:

Reviewer: ```suggestion
To get an existing subscription, use the following GET request:
```

> To get existing subscriptions, do the following GET request on your topic:

Reviewer: ```suggestion
To get existing subscriptions, use the following GET request on your topic:
```

`GET /streams/subscribers/webhook/api/v1/topics/{topicId}/subscriptions`

> See [pagination](/docs/topics-api/#pagination) to get more information about how pagination and sorting works.

Reviewer: ```suggestion
See [pagination](/docs/topics-api/#pagination) for more information on how pagination and sorting works.
```
